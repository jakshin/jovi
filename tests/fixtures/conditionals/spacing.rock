(The traditional way of expressing "else if" is to nest an "if" under an "else")

if false
  say "you won't see this"
else
  if true
    say "you will see this"
  else
    say "you won't see this"
    (end else)
  (end else)
say "you should see this"

(But if you put "else if" one one line, you need fewer blank lines to close blocks)

if false
  say "you won't see this either"
else if true
  say "you will see this too"
else
  say "you won't see this either"
  (end else)
say "you should see this too"
