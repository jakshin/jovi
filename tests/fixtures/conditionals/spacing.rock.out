you will see this
you should see this
you will see this too
you should see this too
